airgun/entities/computeresource.py

```diff
@@ -75,6 +76,17 @@ def vm_poweroff(self, rhev_name, vm_name):
         if vm['Power'].widget.read() == 'On':
             vm['Actions'].widget.click(handle_alert=True)
 
+    def vm_import(self, rhev_name, vm_name, hostgroup, location):
+        """Imports the specified VM"""
+        view = self.navigate_to(self, 'Detail', rhev_name=rhev_name)
+        view.virtual_machines.search(vm_name)
+        view.virtual_machines.more_button.click()
+        view.virtual_machines.import_button.click()
+        # we are already at import view
+        view = self.navigate_to(self, 'Import', rhev_name=rhev_name)
```

Comment:
`self.navigate_to(HostEntity, 'Create')`

Reply:
Done. Cool, I wasn't sure if this is a good idea.